src/lowlevel/Video.cpp

```diff
@@ -896,6 +896,35 @@ Rectangle Video::get_viewport() {
 
 /**
  * \brief Converts window coordinates to quest size coordinates.
```

Comment:
The function below doesn't return anything if the point in not in the viewport, this one does.